worker/setup.py

```diff
@@ -1,42 +0,0 @@
-from setuptools import setup, find_packages
-
-# should match codalab/common.py#CODALAB_VERSION
-CODALAB_VERSION = "0.4.0"
-
-
-def get_requirements(*requirements_file_paths):
-    requirements = []
-    for requirements_file_path in requirements_file_paths:
-        with open(requirements_file_path) as requirements_file:
-            for line in requirements_file:
-                if (
-                    line[0:2] != '-r'
-                    and line.find('git') == -1
-                    and line.find('codalabworker') == -1
-                ):
-                    requirements.append(line)
-    return requirements
-
-
-setup(
```

Comment:
Should we publish some kind of notice to pypi that codalabworker is deprecated and should be replaced by codalab?